engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -418,20 +425,18 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
 
   private def validateCwlNamespace(source: WorkflowSourceFilesCollection,
                                    workflowOptions: WorkflowOptions,
-                                   pathBuilders: List[PathBuilder]): ErrorOr[ValidatedWomNamespace] = {
+                                   pathBuilders: List[PathBuilder]): Parse[ValidatedWomNamespace] = {
     // TODO WOM: CwlDecoder takes a file so write it to disk for now
     import better.files._
-    import cats.syntax.either._
 
     val cwlFile = File.newTemporaryFile(prefix = workflowIdForLogging.toString).write(source.workflowSource)
 
     try {
       for {
-        wf <- CwlDecoder.decodeAllCwl(cwlFile).map {
-          _.select[Workflow].get
-        }.value.unsafeRunSync.toValidated
-        executable <- wf.womExecutable
-        graph <- executable.graph
+        cwl <- CwlDecoder.decodeAllCwl(cwlFile)
+        wf <- EitherT{ IO{cwl.select[Workflow].toRight(NonEmptyList.one(s"expected a workflow but got a $cwl"))}}
+        executable <- EitherT{IO {wf.womExecutable}}
```

Comment:
I apologize as I know you hate these sorts of comments, and yes this is an argument for scalafmt but I hate when sometimes it is "{ I} and then "{I" in other places. I have an opinion on which I prefer but it's really the back and forth which jumps out at me

Reply:
I totally agree, my point w/ scalafmt is that I want the computer to do it for me.  I'll fix it.

Reply:
yeah that's why i try to not make this comment every time i see it, i feel a little bad when i do :)